.revdeprefs.yaml

```
@@ -0,0 +1,6 @@
+- insightsengineering/tern
+- insightsengineering/tern.gee
+- insightsengineering/tern.mmrm
+- insightsengineering/tern.rbmi
+- insightsengineering/scda.test@remotes_tern
```

```suggestion
- insightsengineering/scda.test
```
update this line, if https://github.com/insightsengineering/scda.test/pull/171 is merged

---

I think it would be better to remove Remotes field from `tern.rbmi`. WDYT?

---

hi @pawelru , if we can merge this in https://github.com/insightsengineering/scda.test/pull/171, i think we can change it to  insightsengineering/scda.test